scripts/run_with_docker.sh

```diff
@@ -14,6 +14,8 @@ docker run -it --rm \
   -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
   -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
   -e SQLALCHEMY_DATABASE_URI=$SQLALCHEMY_DATABASE_URI \
+  -e REDIS_ENABLED=${REDIS_ENABLED:-0} \
```

Comment:
I don't think we would need to do this if we are editing the app/config.py file directly (see my comment above)? But this may not be the case if we are using an environment variable instead to control whether we use redis or not.

Reply:
Indeed, we still need this as I'm not proposing someone have to go and edit a file. However, unless the environment variable is set you'll get an error, hence the default of "0" here.